README.md

```diff
@@ -1,24 +1,160 @@
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+**To get a local copy up and running, follow these steps.**
+
+1. Download or clone this [repostory](git@github.com:omwomo/hello-rails-back-end.git).
+2. Provide a modern web browser.
+
+### Prerequisites
+
+**In order to run this project you need:**
+
+- ✔ Ruby installed in your machine. you can download it from [here](https://www.ruby-lang.org/en/downloads/)
+- ✔ IDE or a code editor installed in your machine.
+- ✔ IRB.
+- ✔ Get Postgresql up and running. you can download it from [here](https://www.postgresql.org/download/windows/).
+- ✔ [Git](https://git-scm.com/downloads) installed in your machine.
+- ✔ Sign in or sign up to your [Github](https://github.com/) account.
+- ✔ A professional editer such as [VS Code](https://code.visualstudio.com/download).
+- ✔ An Updated web browser such as Google Chrome, you can download it from [here](https://www.google.com/chrome/).
+
+### Setup
+
+- Clone this [repository](git@github.com:omwomo/hello-rails-back-end.git) to your desired folder:
+
+- Run this command in your command line interface:
+
+```sh
+  cd [YOUR FOLDER]
+  git git@github.com:omwomo/hello-rails-back-end.git
+  cd hello-rails-back-end
+  rails db:seed
+
+```
+
+- Update the database.yml with your database credentials under the development section
+
+
+### Usage
+
+- To run the project, execute the following command:
+
+```sh
+   rails server
+```
+
+### Deployment
+
+**This project is deployed by the author, no permission for deployment by any other client.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Eugene Oduor**
+
+- GitHub: [@omwomo](https://github.com/omwomo)
+- LinkedIn: [@omwomo](https://www.linkedin.com/in/omwomo/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add info about each category**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/omwomo/hello-rails-back-end/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly drop a start ⭐️ for the [repository](git@github.com:omwomo/hello-rails-back-end.git);
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my heartfelt gratitude to **Microvere** for the invaluable learning experience they have provided.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Your license link is currently broken, please kindly fix this so that your project can be approved :+1: :sparkles: